examples/MyModule/step_pattern_builder.cpp

```diff
@@ -101,10 +101,7 @@ mynest::StepPatternBuilder::connect_()
 // This code is based on AllToAllBuilder, except that we step
 // by source_step_ and target_step_ except for stepping by 1.
 
-#pragma omp parallel
-  {
-    // get thread id
-    const int tid = nest::kernel().vp_manager.get_thread_id();
+  NEST_PARALLEL_FOR_THREAD(tid)
```

Comment:
I think that for the sense of code clarity, we should include the parallel section in explicit curly braces, even though they are superfluous. This will make the code more readable. Example:
```c++
NEST_PARALLEL_FOR_THREAD( thread_id )
{
  <code>
}
NEST_PARALLEL_END
```
I would also, as in the example here, suggest to use `thread_id` as variable name for the thread throughout.